config/production.yml

```diff
@@ -0,0 +1,12 @@
+proxy:
+  host: naomi.dide.ic.ac.uk
+  port_http: 80
+  port_https: 443
+  ssl:
+    certificate: VAULT:secret/hint/ssl/naomi:certificate
```

Comment:
Is the idea with this path that 'hint' is a general category, and 'naomi' is this particular instance?

Reply:
yes - if we need to do any faff with the certs for the unaids address later